modules/openapi-generator/src/main/resources/cpp-pistache-server/model-header.mustache

```diff
@@ -51,6 +43,8 @@ public:
     {{/required}}
     {{/vars}}
 
+    friend void to_json(nlohmann::json& j, const {{classname}}& o);
```

Comment:
These functions are not accessible? how will we convert the json object to this model

Reply:
[Documentation](https://github.com/nlohmann/json#arbitrary-types-conversions) shows how to convert to and from arbitrary types using an external function in the same namespace. That external function needs access to private members of the class to convert efficiently, so it needs to be a friend function.

Reply:
Hmm I was thinking if I have to convert the resquest to an class model how will I do that?

Reply:
You can do
`Object obj = nlohmann::json::parse(string).get<Object>();`
or if you already have the json object
`Object obj = json.get<Object>();`
The other way is
`string str = nlohmann::json(obj).dump();`
or
`nlohmann::json json = obj;`